gitlab-runner/Dockerfile

```diff
@@ -0,0 +1,81 @@
+FROM ubuntu:16.04
+
+ENV GCLOUD_SDK_VERSION=200.0.0
+ENV GCLOUD_SDK_URL=https://dl.google.com/dl/cloudsdk/channels/rapid/downloads/google-cloud-sdk-${GCLOUD_SDK_VERSION}-linux-x86_64.tar.gz
+ENV PATH="${PATH}:/opt/google-cloud-sdk/bin"
+ENV TERM=xterm
+
+ARG ANSIBLE_VERSION=2.6.2~ubuntu16.04.1
+ARG KOPS_VERSION=1.9.0
+ARG KUBECTL_VERSION=1.11.0
```

Comment:
Can we go w/ 10 - for now. Waiting for some `kubectl set env` fixes to be available in 1.11+. However, this may need a test run of a pipeline w/ at least 1-2 jobs so that we can be sure it wont break existing scripts.